lib/assets/block-navigation.js

```diff
@@ -1,63 +1,78 @@
-var jumpToCode = (function init () {
-  // Classes of code we would like to highlight
-  var missingCoverageClasses = [ '.cbranch-no', '.cstat-no', '.fstat-no' ];
+var jumpToCode = (function init() {
+  // Classes of code we would like to highlight in the file view
+  var missingCoverageClasses = ['.cbranch-no', '.cstat-no', '.fstat-no'];
```

Comment:
Here the indentation looks to be 2 spaces, whereas further down they are 4. If it’s not a huge effort, it’d be good to be consistent.